prompting/validators/tasks.py

```diff
@@ -122,24 +124,61 @@ def compose_prompt(self) -> str:
 
 
 def create_summarization_task(base_text: str) -> SummaryTask:
-    possible_criterias = [
-        MatchLengthCriteria(
-            penalty=0.1,
-            target_length=random.randint(50, 200),
-            unit=TextLengthUnitEnum.WORDS,
-        ),
-        MatchLengthCriteria(
-            penalty=0.1,
-            target_length=random.randint(4, 8),
-            unit=TextLengthUnitEnum.SENTENCES,
-        ),
-    ]
+    # scope 1: bullet points, scope 2: numbered list, scope 3: simple layout
+    scope = random.randint(1, 3)
+
+    select_bullet_point_layout = scope == 1
+    select_numbered_list_layout = scope == 2
+
+    # scope 1 or 2: define criteria set for bullet points or numbered list
+    if select_bullet_point_layout or select_numbered_list_layout:
+        if select_bullet_point_layout:
+            layout_criteria = MatchLayoutCriteria(
+                layout_type=LayoutMatchTypeEnum.UNORDERED_LIST,
+                penalty=0.5,
+                text="Your response should be ordered in format of bullet points.",
+            )
+        else:
+            layout_criteria = MatchLayoutCriteria(
+                layout_type=LayoutMatchTypeEnum.NUMBERED_LIST,
+                penalty=0.5,
```

Comment:
no text here?

Reply:
it will get the default text `Your response should be ordered in format of numbered list`. The bullet point criteria would list like "unordered list", that's why I override it to be "bullet points"